cpp/src/slice2cs/Gen.cpp

```diff
@@ -1094,35 +1078,15 @@ Slice::CsVisitor::requiresDataMemberInitializers(const DataMemberList& members)
 }
 
 void
-Slice::CsVisitor::writeDataMemberInitializers(const DataMemberList& members, const string& ns, unsigned int baseTypes)
+Slice::CsVisitor::writeDataMemberInitializers(const DataMemberList& members, unsigned int baseTypes)
 {
+    // Generates "= new()" for each struct field mapped to a class.
```

Comment:
We can't use field/property default values for this logic.

It's unfortunate we do that, but we can't change it without breaking backwards compatibility.